vm/src/syscalls.rs

```diff
@@ -16,8 +15,27 @@ pub struct Syscalls {
     log_buffer: Vec<Vec<u8>>,
     input: VecDeque<u8>,
     output: Vec<u8>,
+    label: Vec<Vec<u8>>,
 }
 
+pub enum SyscallCode {
+    WriteLog = 1,
+    ReadFromPrivateInput = 2,
+    WriteToOutput = 3,
+    ProfileCycles = 4,
```

Comment:
If you change this to `5` as noted you should also put a comment here explaining the skip of `4`.

Reply:
updated in https://github.com/nexus-xyz/nexus-zkvm/pull/243/commits/25013349658e9b04816cee6f148b723cd4bae97b